python/ray/dag/dag_node_operation.py

```diff
@@ -195,6 +189,30 @@ def _add_edge(from_node: _DAGOperationGraphNode, to_node: _DAGOperationGraphNode
     to_node.in_edges.add((from_node.task_idx, from_node.operation.type))
 
 
+def _update_candidate_nodes(
```

Comment:
```suggestion
def _push_candidate_node_if_ready(
```

(more accurate function name)

Reply:
Done.